personal-website/content/about.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!-- Header -->
+    <head>
+        <meta charset="UTF-8">
+        <script src="https://use.fontawesome.com/c0ac3b38ec.js"></script> <!--insert Linkedin and Github icons as classes --> 
+        <title>Saharai's Personal Portfolio WebPage</title>
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <link rel="stylesheet" type="text/css" href="/Users/ada/Developer/projects/personal-website/content/stylesheets/style_bio.css">
```

Comment:
Be careful about using absolute path, this will work on your own machine but when you push your code on a project you're working with others they won't have the same file path as you on their machines.

I recommend using ***relative*** path which describes the path starting from the root of your project. So in this case `./stylesheets/style.css` which we can verbalize into:

>  "from the current directory I'm in `.` get a resource in the `/stylesheets/` folders called `style.css`"